packages/libs/react-ui/src/components/Form/Select/Select.css.ts

```diff
@@ -1,47 +1,40 @@
-import { style } from '@vanilla-extract/css';
-import { atoms, bodyBaseRegular, ellipsis, token } from '../../../styles';
+import { globalStyle, style, styleVariants } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+import { atoms, ellipsis, token } from '../../../styles';
+import { inputFontTypeVariants, inputSizeCompoundVariants } from '../Form.css';
+
+export const selectButtonValue = atoms({
+  display: 'flex',
+  flex: 1,
+});
 
 export const selectButtonClass = style([
-  bodyBaseRegular,
   atoms({
     display: 'flex',
-    alignItems: 'center',
-    borderRadius: 'sm',
     backgroundColor: 'input.default',
-    border: 'none',
+    justifyContent: 'space-between',
     color: 'text.base.default',
-    outline: 'none',
     flex: 1,
     overflow: 'hidden',
+    position: 'relative',
+    alignItems: 'center',
   }),
   {
+    paddingBlock: '0',
     paddingInlineStart: token('spacing.md'),
-    paddingInlineEnd: token('spacing.md'),
-    paddingBlock: token('spacing.sm'),
-    boxShadow: `0px 1px 0 0 ${token('color.border.base.default')}`,
-    outlineOffset: '2px',
+    paddingInlineEnd: '0',
     selectors: {
       '&[data-hovered]': {
         cursor: 'pointer',
       },
-      '&[data-positive]': {
-        outline: `2px solid ${token('color.border.semantic.positive.@focus')}`,
-      },
-      '&[data-disabled]': {
-        pointerEvents: 'none',
-        backgroundColor: token('color.background.input.inverse.default'),
-        color: token('color.text.base.inverse.@disabled'),
-      },
-      '&[data-focused]': {
-        outline: `2px solid ${token('color.border.semantic.info.@focus')}`,
-      },
-      '&[data-invalid]': {
-        outline: `2px solid ${token('color.border.semantic.negative.@focus')}`,
-      },
     },
   },
 ]);
 
+globalStyle(`${selectButtonClass} > span`, {
```

Comment:
do we want a central place for all global changes even when its scoped to our class? Just a thought

Reply:
I would say no, this is heavily reliant on context. If we would move this to a separate file then you would never know that it's impacting your component without a proper search.